src/components/Query.vue

```diff
@@ -1,48 +1,89 @@
 <template>
     <div>
-        <v-text-field
-            solo
-            autocomplete="off"
-            label="Enter Search Query"
-            v-model="query"
-            v-on:keyup.enter="queryStackExchange"
-            id="input-search"
-            append-icon="search"
-            @click:append="queryStackExchange"
-            >
-        </v-text-field>
-        <questions :results=results></questions>
+        <div class="query">
+            <v-row>
+                <!-- query -->
+                <v-col class="col-query">
+                    <v-text-field class="input-query"
+                        v-bind:class="{'theme--dark': darkMode}"
+                        filled
+                        dense
+                        autocomplete="off"
+                        placeholder="Enter Search Query"
+                        v-model="query"
+                        v-on:keyup.enter="queryStackExchange"
+                        id="input-search"
+                        append-icon="mdi-magnify"
+                        @click:append="queryStackExchange"
+                        >
+                    </v-text-field>
+                </v-col>
+                    <!-- tags -->
+                    <v-col class="col-tags">
+                        <v-text-field class="input-tags"
+                            v-bind:class="{'theme--dark': darkMode}"
+                            filled
+                            dense
+                            autocomplete="off"
```

Comment:
Quick question - was this story supposed to include the tag API to allow for autocomplete?

Reply:
I think we discussed about it. I'll look into it. Thanks!